cmd/bbolt/command_info.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+
+	bolt "go.etcd.io/bbolt"
+)
+
+func newInfoCommand() *cobra.Command {
+	infoCmd := &cobra.Command{
+		Use:   "info PATH",
+		Short: "Info prints basic information about the Bolt database at PATH.",
+		RunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
Suggest validating args cobra style, if I don't supply the DB path currently and just run `bbolt info` all I get is a panic and it's unclear from a user perspective what was done wrong. 

Example arg validation:

https://github.com/etcd-io/bbolt/blob/cd4a7570d562c4f57ef112981b4140fd8ddc6fda/cmd/bbolt/command_surgery.go#L57-L65

Reply:
I think this can use
```
Args: cobra.ExactArgs(1)
```

but I've use the example that you list